storage/bucket.go

```diff
@@ -945,6 +945,82 @@ func (b *BucketAttrs) toProtoBucket() *storagepb.Bucket {
 	}
 }
 
+func (ua *BucketAttrsToUpdate) toProtoBucket() *storagepb.Bucket {
+	if ua == nil {
+		return &storagepb.Bucket{}
+	}
+
+	var labels map[string]string
```

Comment:
As discussed, the label keys will need to go into the updateMask as well (and the functionality isn't actually working). Let's leave this as a TODO.